### ISSUE TITLE
Encyclopaedia Aethiopica Gǝʿǝz and Amharic transliteration system

### ISSUE THREAD

**Issue by ronaldtse** [interscript/interscript-ruby#391]
https://brill.com/view/book/edcoll/9789004419582/front-11.xml?language=en

![image](https://user-images.githubusercontent.com/11865/92435530-30f65700-f1d5-11ea-912b-a497ab7bcc84.jpeg)
![image](https://user-images.githubusercontent.com/11865/92435534-3358b100-f1d5-11ea-86a0-bfcd19da609e.jpeg)
![image](https://user-images.githubusercontent.com/11865/92435537-35227480-f1d5-11ea-8b59-d3313faa096c.jpeg)

**Comment by ronaldtse** [comment]
For @tsega

**Comment by tsega** [comment]
@ronaldtse the reference document does not have the metadata as the others. The file name scheme is not clear as well.

**Comment by ronaldtse** [comment]
The Interscript system format follows CC/ISO 24229: https://calconnect.gitlab.io/tc-localization/csd-transcription-systems/documents/iso-24229.html

i.e.
1. Authority name: let's use `var` here because this is a book
2. ISO 639 language code: this is `amh` and `get`.
3. ISO 15924 source and target script codes: this is `Ethi` to `Latn`
4. Some unique ID: let's use `eae-2003` as Wikipedia abbreviates it as `EAe` (https://en.wikipedia.org/wiki/Encyclopaedia_Aethiopica)

So we could use `var-amh-Ethi-Latn-eae-2003` and `var-gez-Ethi-Latn-eae-2003` for this system (the first implements it, the second "inherits" the first map).

**Comment by tsega** [comment]
@ronaldtse been working on these for a while now. They have been particularly difficult to implement as the diacritical marks are not easy to get.

Right now, some tests are failing, for example:

```
- source: እንጠብቅሻለን አለብን አደራ
    expected:  ̓ənəṭäbəqəšalänə  ̓aläbənə  ̓adära
```

fails

```
expected: "̓ənəṭäbəqəšalänə  ̓aläbənə  ̓adära"
          got: " ̓ənəṭäbəqəšalänə  ̓aläbənə  ̓adära"
```

Do you know how to fix this?

**Comment by ronaldtse** [comment]
Perhaps @bilashsaha could help with the diacritics?

**Comment by tsega** [comment]
Alright, figured it out. It was the mark I used for two sets of characters.

I was using `U+02BC ʼ ` and `U+02BD ʽ ` as opposed to `U+02BE ʾ ` and `U+02BF  ʿ ` respectively.

Will raise a PR after rebasing.

**Comment by tsega** [comment]
**PR** - https://github.com/interscript/interscript/pull/606

**Summary**
- Add `var-amh-Ethi-Latn-eae-2003` and `var-gez-Ethi-Latn-eae-2003`

**Comment by ronaldtse** [comment]
Closed as #606 has been merged. Thank you @tsega !
